remove/remove.go

```diff
@@ -105,14 +105,24 @@ func checkWriteAccess(modulePath, scriptPath string) error {
 	return nil
 }
 
-func removeLocalFiles(modulePath, scriptPath string) error {
-	if err := removeAllNoDescend(modulePath); err != nil {
+func removeLocalFiles(modulePath, version, scriptPath string) error {
+	if err := removeAndParentIfEmpty(modulePath, version); err != nil {
 		return err
 	}
 
 	return removeAllNoDescend(scriptPath)
 }
 
+func removeAndParentIfEmpty(modulePath, version string) error {
+	if err := os.Remove(filepath.Join(modulePath, version)); err != nil {
+		return err
+	}
+
+	os.Remove(modulePath)
```

Comment:
linter should have complained about this. In any case, for clarity there should be a comment (eg. `# nolint:errcheck`) to explicitly say we don't care about the error.

Reply:
The linter actually complains when I add the nolint directive.